addons/website_sale/static/src/scss/website_sale.scss

```diff
@@ -154,6 +154,10 @@ $o-wsale-products-layout-grid-gutter-width: min($grid-gutter-width / 2, $o-wsale
 
     #o_wsale_offcanvas_content {
         @include o-wsale-scrollbar-subdle();
+
+        .list-group-item {
```

Comment:
I moved these changes in `bootstrap_overridden.scss` and made it global for all `.list-group` as the default background is hardcoded `white` and color is hardcoded `gray-900`. If we plan to make the frontend dark-mode compatible, this will be needed.